css-layout-classes/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Layout Classes</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <h1>Choose your character!</h1>
+      <div class="row">
+        <div class="column">
+
+        </div>
+      </div>
+      <div class="row">
+        <div class="column">
+          <img src="images/link.png" alt="Link">
+        </div>
+        <div class="column">
+          <h2>Super Smash Bros Ultimate</h2>
+          <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quos beatae nesciunt in incidunt modi nisi molestias rem aspernatur repellat odit voluptate fugit maxime aperiam libero non, quo voluptatibus eum repudiandae!</p>
+        </div>
+        <div class="column">
+          <img src="images/yoshi.png" alt="Yoshi">
+        </div>
+      </div>
+
+
+        <div class="row">
+          <div class="column-quarter">
+            <img src="images/kirby.png" alt="Kirby">
+          </div>
+          <div class="column-second-half">
+          <img src="images/fox.png" alt="Fox">
+          </div>
+          <div class="column-three-fourths">
+            <img src="images/pikachu.png" alt="Pikachu">
+          </div>
+          <div class="column-fourth">
+            <img src="images/ness.png" alt="Ness">
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-half">
+            <h2>Donkey Kong</h2>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-two-thirds">
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati atque tenetur aperiam voluptate deserunt praesentium architecto assumenda incidunt modi. Doloribus adipisci dolorem magni quis minima, architecto repudiandae eius. Voluptate, dolor.</p>
+
+            <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nostrum dolorum rem expedita molestias natus ipsum temporibus dolore, nobis vel asperiores similique animi itaque dolores quae cumque cum debitis, ea quia?</p>
+          </div>
+          <div class="column-right-d">
+            <img src="images/dk.png" alt="Donkey Kong">
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-first">
+            <img src="images/mario.png" alt="Mario">
+          </div>
+          <div class="column-middle">
+            <h2>Mario</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Blanditiis sint nemo accusamus totam ipsa eaque tempora mollitia iure. Voluptatum ratione consequuntur consequatur eos architecto velit corporis accusamus harum iure nostrum.</p>
+          </div>
```

Comment:
These column classes still don't describe their behavior. What is their size? Name the class after that. `column-one-third`, `column-two-thirds`, etc.